tests/test_ui.txt

```diff
@@ -89,7 +89,7 @@ Default CLI options, passed to ui.main():
 ...         return ks
 ...
 ...     with patch.object(term, "inkey", new=inkey):
-...         ui.main(term, dataobj, host, options, "",
+...         ui.main(term, dataobj, host, options,
```

Comment:
It seems we're still using optparse in this test (see above, out of diff context).

In fact, I believe we should get rid of argparse/optparse when entering application code from the `cli` module. This can be done by transforming `argparse.Namespace` into a `dict` calling `ui.main()` with `vars(args)`. This change is probably more invasive, so I'd suggest to do it in a dedicated commit.

Reply:
yes ... completely missed this part ... should have waited monday to push ...

Reply:
I changed it but I am not sure I did it right.

Reply:
Improved this in the commit I just pushed. (Basically use an `argparse.Namespace` object directly. The overall diff is smaller.)

Reply:
yes ... indeed. I had the feeling I was writing way to much code for this ...

Reply:
thanks.